MARS_QA/Mars_Pages/ProfileDescription.cs

```diff
@@ -0,0 +1,45 @@
+﻿using OpenQA.Selenium;
+using MARS_QA.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MARS_QA.Mars_Pages
+{
+    internal class ProfileDescription : CommonDriver
```

Comment:
can you please implement the POM pattern! please see 
https://docs.specflow.org/projects/specflow/en/latest/ui-automation/Selenium-with-Page-Object-Pattern.html

Reply:
please fix this for all pages